utils/handy.ts

```diff
@@ -21,8 +22,9 @@ export async function IPify(): Promise<string> {
  */
 export function searchParamsSerializer(url: URL): Record<string, string> {
   const params: Record<string, string> = {};
-  for (const [key, value] of url.searchParams.entries()) {
+  // Convert the entries to an array and iterate over it
+  Array.from(url.searchParams.entries()).forEach(([key, value]) => {
```

Comment:
Did you test it ??

Reply:
i found same issue with this solution in other place you can test it and see if it work correctly...

Reply:
No that cool I'll be implementing it soon

Reply:
i fix it, using direct push to the main:
https://github.com/handyscript/handyutility/blob/4b7f38ec4546c8d10034c269e0a0a319706a0471/utils/handy.ts#L24

Reply:
Great JOB, Keep going Bro.